packages/react-utilities/src/utils/on.ts

```diff
@@ -0,0 +1,9 @@
+export const on = <T>(
```

Comment:
I know I was the one who suggested making this shared, but I actually wonder if there should be a different solution for converged to make cleanup easier. Would be interesting to briefly discuss that with the rest of the team.

(Also if we do keep this, it needs a doc comment.)

Reply:
After a discussion in the tech sync we decided that there are better ways to go then the `on` utility in converged. I'm removing it from this PR and starting to look into alternatives such as a `useEvent` hook for the Slider and Switch components.